src/components/editor/Editor.tsx

```diff
@@ -5,74 +5,62 @@ import {selectEditorState} from 'store/selectors';
 
 import {AnnotationPriorities, annotationPrioritiesLabels} from '../consts';
 import styles from './Editor.scss';
+import {DragTracker, IVec2} from '../../utils/move';
 import PriorityButton from './priority-button/PriorityButton';
 import {Modal, Popup} from 'semantic-ui-react';
-import {DraggableWidget} from 'components/widget';
+import Widget from '../widget/Widget';
+import {IEditorState, IEditorProps, IEditorForm} from './interfaces';
+import { DraggableWidget } from 'components/widget';
+import { hideEditor } from 'store/widgets/actions';
```

Comment:
this should be importable from `'store/actions'`, along with the one below

Reply:
Thanks, I missed this particular "shortcut" pattern you made.

Reply:
The `store/actions.ts` file just re-exports stuff from internal actions.ts files, similarly `store/selectors.ts` 🙂

Reply:
Yes, that's what I meant by "shortcut" :) I just didn't notice this pattern before.